test/network/BancorNetwork.ts

```diff
@@ -2129,42 +2133,26 @@ describe('BancorNetwork', () => {
 
             value ||= sourceTokenAddress === NATIVE_TOKEN_ADDRESS ? amount : BigNumber.from(0);
 
-            let retVal = BigNumber.from(0);
-            if (getRetVal) {
-                retVal = await network
-                    .connect(trader)
-                    .callStatic.tradeBySourceAmount(
-                        sourceTokenAddress,
-                        targetTokenAddress,
-                        amount,
-                        minReturnAmount,
-                        deadline,
-                        beneficiary,
-                        {
-                            value
-                        }
-                    );
-            }
+            const caller = callStatic ? network.connect(trader).callStatic : network.connect(trader);
```

Comment:
Maybe `caller` is not the best name. I'd rather use something like `method` (since it's basically a method chain)?